foundry.toml

```diff
@@ -2,7 +2,7 @@
 src = 'src'
 out = 'out'
 libs = ['lib']
-solc_version = '0.8.19'
+solc_version = '0.8.21'
```

Comment:
if it is not required to use 0.8.21, let's stick with 0.8.19 since there are some L2 that does not support PUSH0 (which is default from 0.8.20 if i remember it correctly)

Reply:
You actually use the latest solidity version without targeting `shangai` evm version (that's the version with push0), just have to specify `evm_version = 'paris'` in the foundry config